tests/basictests/dsp-operator.sh

```diff
@@ -208,12 +208,44 @@ function delete_pipeline() {
     os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' -X DELETE https://${ROUTE}/apis/v1beta1/pipelines/${PIPELINE_ID} | jq" "" $odhdefaulttimeout $odhdefaultinterval
 }
 
+function create_recurring_run() {
+    header "Creating the Recurring Run from uploaded pipeline"
+
+    JOB_ID=$((curl -k -H "Authorization: Bearer ${SA_TOKEN}" \
+        -H "Content-Type: application/json" \
+        -X POST "${ROUTE}/apis/v1beta1/jobs" \
+        -d @- << EOF
+        {
+            "name":"test-recurring-run",
+            "pipeline_spec":{
+                "pipeline_id":"${PIPELINE_ID}"
+            },
+            "max_concurrency": 10,
+            "trigger": {
+                "periodic_schedule": {
+                    "interval_second": 3600
+                }
+            },
+            "enabled": true
+        }
+EOF
+        ) | jq -r .id)
+
+    os::cmd::try_until_not_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/jobs/${JOB_ID} | jq '" "null" $odhdefaulttimeout $odhdefaultinterval
+}
+
+function verify_recurring_run_availabilty() {
+    header "verify the Recurring Run exists"
+
+    os::cmd::try_until_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/jobs | jq '.total_size'" "1" $odhdefaulttimeout $odhdefaultinterval
+}
+
 
 echo "Testing Data Science Pipelines Operator functionality"
 
 verify_data_science_pipelines_operator_install
 create_and_verify_data_science_pipelines_resources
-check_custom_resource_conditions
+#check_custom_resource_conditions
```

Comment:
Why are we commenting out this function?

Reply:
Good catch! It should be uncommented. Let me fix that.